tasks/tasks.py

```diff
@@ -110,7 +110,9 @@ def upload_artifacts(self):
                             timeout=5*60))
         except Exception as exc:
             logging.debug(exc, exc_info=True)
-            raise TaskException(self, "Failed to publish artifacts")
+            msg = 'Failed to publish artifacts from {hostname}'.format(
+                hostname=socket.gethostname())
```

Comment:
This could leak the FQDN, as pointed by [some comments](https://github.com/freeipa/freeipa-pr-ci/search?q=socket.gethostname&unscoped_q=socket.gethostname) I've found in the codebase: https://github.com/freeipa/freeipa-pr-ci/blob/81e6d9fa91e6acb18eb721eabd4c79c39320257d/tasks/remote_storage.py#L223-L224

Reply:
you right!, thanks Armando, Will amend the commit.